dask/array/tests/test_routines.py

```diff
@@ -2455,6 +2455,42 @@ def test_einsum(einsum_signature):
         )
 
 
+def test_einsum_chunksizes():
```

Comment:
@phofl I was just looking at test runtimes and this one adds >70s per CI job. That's a little excessive. I haven't checked but suspect this is because there are so many dimensions here. Is this test actually requiring this many dimensions or can it be shrunk a little?

Reply:
Yikes this is bad, it was like 5s locally :(

I can shrink it, yes